### DISCUSSION TITLE
Pending verification with "We need more information to continue."

### DISCUSSION THREAD

**Discussion by slabua** [community/community#110871] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

All steps are completed, not sure what more information are required to continue.
Is there some problem with my request?
Thanks in advance.
![image](https://github.com/community/community/assets/1002978/6cf5149a-edbe-49f4-943d-c6bb972f670c)

**Comment by thilinaabey** [comment]
Cancel the approval request and reapply

**Comment by probablynotnmp** [comment, marked as answer]
Try reaching out to GitHub support directly for clarification on what specific information or steps are needed to proceed with your sponsorship application and if that doesn't help try to cancel the approval request for now then just reapply for it.

Hope it helps :)

**Comment by donzov** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> All steps are completed, not sure what more information are required to continue. Is there some problem with my request? Thanks in advance. ![image](https://private-user-images.githubusercontent.com/1002978/309711628-6cf5149a-edbe-49f4-943d-c6bb972f670c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5NjMxOTksIm5iZiI6MTcxMDk2Mjg5OSwicGF0aCI6Ii8xMDAyOTc4LzMwOTcxMTYyOC02Y2Y1MTQ5YS1lZGJlLTQ5ZjQtOTQzZC1jNmJiOTcyZjY3MGMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjBUMTkyODE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjkwZGNjNDAwYzViOTU3MmFlOWY3OTJlYWFmZGJiNWZiNDAxYmI4Nzc3YzljNzBlYmRjNmMxMDQ2NDJhMGJhMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.LoYEb3On40VesHCqINxDO4dZ5ObKWkyj6KcacfKjZ8o)
